### ISSUE TITLE
Fractional land/water mask does only produce 0 and 100 values

### ISSUE THREAD

**Issue by JanWev** [bcdev/snap-idepix#59]
The fractional land/water mask produces only 0 or 100 values independent from selected resolution and subsampling factors. At coastal zones there should be values between 0 and 100. But these values do not exist. Possible bug.

**Comment by JanWev** [comment]
Grit and I realized that this is caused by the subpixel accuracy, which was not selected. It would be good to have the sub-pixel accuracy set by default in SNAP.

**Comment by marpet** [comment]
Yes, this is true. This is the behaviour of the Land/water mask since the beginning.
I have created an issue in SNAP for this: https://senbox.atlassian.net/browse/SNAP-1515
